pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
-addopts = --reuse-db --tb=native
+addopts = --reuse-db --tb=native -m 'not es_tests' -v
```

Comment:
The `-m 'not es_tests'` flag is used by `tox` to break the tests up into several parallel runs, to shorten the runtime of the test suite. It's not something we want as a default.

I also don't think `-v` is a good default. Users can add it when they think it makes sense. I don't think we should make that choice for them.